docs/docker_deployment_guide.md

```diff
@@ -115,23 +115,23 @@ docker compose exec php bin/console kbin:user:admin <username>
 docker compose exec php bin/console kbin:ap:keys:update
 ```
 
-Next, log in and create a magazine named "random" to which unclassified content from the fediverse will flow.
+Next, log in and create a magazine named “random” to which unclassified content from the Fediverse will flow.
 
 ### Add auxiliary containers to `docker-compose.yml`
 
-Add any auxiliary container as you want. For example, add a Nginx container as reverse proxy to provide HTTPS encryption.
+Add any auxiliary container as you want. For example, add a Nginx container as a reverse proxy to provide HTTPS encryption.
```

Comment:
"Nginx" is a strange word when it comes to adding an indefinite article in front of it. One might think "a" would be correct since the word starts with "n", but... when I read through this sentence out loud, I naturally correct "a" to "an" because "Nginx" is pronounced "engine x" and not "n-jinx".